docs/bis/ec/ec-format.md

```diff
@@ -157,6 +162,24 @@ This format specifies only one unit, decimal formatting and not label override.
         ]
       }
 ```
+This format specifies units which have labels that are intentionally missing or set to an empty string.
+
+```json
+    "MyMetersFormat": {
+      "schemaItemType": "Format",
+      "label": "real",
+      "type": "Decimal",
+      "precision": 4,
+      "formatTraits": [
+        "KeepSingleZero",
+        "KeepDecimalPoint",
+        "ShowUnitLabel"
+      ],
+      "composite": {
+        "units": [{ "name": "Units.KM" }, { "name": "Units.M", "label": "" }, { "name": "Units.CM", "label": "metre" }, { "name": "Units.MM", "label": "millimetre" }]
```

Comment:
this has the label for CM as meter ... I know it doesn't really matter but it'd be better if it was correct